exalearn-ui/src/styles.less

```diff
@@ -1 +1,4 @@
 /* You can add global styles to this file, and also import other style files */
+
+html, body { height: 100%; }
```

Comment:
Please, refactor file and make styles definitions multiline:
```
html, body {
    height: 100%;
}
body { 
     margin: 0; font-family: Roboto, "Helvetica Neue", sans-serif; 
}
```

Reply:
my bad